### PR TITLE
ProtocolWidget: Show number of non synced files in tab label.

### PR DESCRIPTION
**Pull request by dragotin**
This pull request adds the number of not synced items to the "not synced" tab

### CODE REVIEW

**Comment by dragotin** [comment]
tab_numbers

**Comment by dragotin** [comment]
@jancborchardt pls.

**Comment by ogoffart** [comment]
The code part looks good to me.

**Comment by jancborchardt** [comment]
I’d just have the number in there in parentheses:

> Not synced (2)

This also saves the plurals etc. – saying »file(s)« is really dirty. ;)

**Comment by ogoffart** [comment]
Normaly the plurals show correctly with the proper translation files

**Comment by dragotin** [comment]
notsynced2
Here we go.

File: src/gui/activitywidget.cpp [hunk 1]
```diff
@@ -556,7 +556,7 @@ void ActivitySettings::slotShowIssueItemCount(int cnt)
 {
     QString cntText = tr("Not Synced");
     if( cnt ) {
-        cntText = tr("Not Synced - %n file(s)", "", cnt);
+        cntText += QLatin1String(" (") + QString::number(cnt) + QLatin1String(")");
```
Comment (ogoffart) [inline src/gui/activitywidget.cpp]: is there not lanquages in which the  parentheses should be put before?

perhaps it's better to keep it in the tr()

**Comment by jancborchardt** [comment]
:+1: 

(In German, it should be »Nicht synchronisiert« with a captalized beginning though, right?)

**Comment by jancborchardt** [comment]
Delete the branch? ;)

**Pull request merged**